setup.cfg

```diff
@@ -181,6 +181,7 @@ dev =
     typing_extensions; python_version < '3.8'
     # Test
     pytest
+    pytest-custom_exit_code  # used in the CI
```

Comment:
Pytest returns exit code 5 if it collects no tests, and in some jobs there are no tests collected. I could simply skip those jobs? Yes, but those combinations don't have tests not because there aren't but because they are misclassified. 

An example: the integration tests job for modeling does not collect any test. That doesn't mean we have no integration tests in the modeling folder, just they're still all mixed up with the unit tests. If I disable the CI job, once the tests get classified properly they won't run. I want to avoid this situation as much as possible, that's why I went for `pytest-custom_exit_code`.

Consider also that, by the time we need to drop this packag, our tests might be in a better shape. One more point for the package: it's [officially recommended by the pytest docs](https://docs.pytest.org/en/7.1.x/reference/exit-codes.html), so the lack of updates might be due to its simplicity.